clients/src/main/resources/common/message/ProduceRequest.json

```diff
@@ -33,7 +33,7 @@
   // Starting in Version 8, response has RecordErrors and ErrorMessage. See KIP-467.
   //
   // Version 9 enables flexible versions.
-  "validVersions": "0-9",
+  "validVersions": "0-10",
```

Comment:
Does it make sense to add a comment about the version bump?

Reply:
yes, forgot to include that.